app/src/main/kotlin/org/eclipse/kuksa/testapp/databroker/vssnodes/viewmodel/VssNodesViewModel.kt

```diff
@@ -41,12 +42,16 @@ class VssNodesViewModel : ViewModel() {
     }
 
     private val vssVehicle = VssVehicle()
-    val vssNodes = listOf(vssVehicle) + vssVehicle.heritage
+    val nodes = listOf(vssVehicle) + vssVehicle.heritage
 
     var node: VssNode by mutableStateOf(vssVehicle)
         private set
 
+    var updateCounter: Int by mutableStateOf(0)
```

Comment:
Needs to be here so we can force an UI update for the "value". Otherwise the VssSignal did not change but the user already typed in another temporary value in the TextField -> Value is not restored